src/assets/src/components/home.tsx

```diff
@@ -22,10 +30,15 @@ function QueueLookup() {
                             placeholder="Queue name or host uniqname..."
                             name="term"
                             value={lookup}
-                            onChange={(e) => setLookup(e.target.value)}
+                            onChange={handleChange}
                         />
-                        <Button type="submit" variant="primary">Search Queues</Button>
+                        <Button type="submit" variant="primary" disabled={emailError}>Search Queues</Button>
                     </InputGroup>
+                    {emailError && (
+                        <Alert variant="warning" className="mt-2">
+                            Emails cannot be used to search queue. Please use uniqname instead.
```

Comment:
The warning block has background of yellow color. Can we use red-font text without background color, to be consistent?

Reply:
Color has been changed: 
<img width="620" alt="Screenshot 2024-04-18 at 10 02 59 PM" src="https://github.com/tl-its-umich-edu/remote-office-hours-queue/assets/56746786/a718f5f4-cab3-455f-be21-2f45a97066c1">